en/option/component/calendar.md

```diff
@@ -120,7 +120,8 @@ Options:
 Calendar coordinates splitLine style.
 
 ### show(boolean) = ${defaultShow|default(true)}
```

Comment:
Somewhat related to this pull request.

I noticed that on some other files the default value for the parameter `defaultShow` is shown on the title of the property, e.g.https://github.com/ecomfe/echarts-doc/blob/gh-pages/en/option/series/graph.md#labelobject-1

Should I open another pull request to fix this or submit an issue?

Reply:
Thanks Jivanvl! What do you mean by `defaultShow` is shown on the title of the property? I didn't get it what is wrong with https://github.com/ecomfe/echarts-doc/blob/gh-pages/en/option/series/graph.md#labelobject-1 .